shotgun_api3/shotgun.py

```diff
@@ -1005,12 +1037,7 @@ def schema_entity_read(self, project_entity=None):
 
         params = {}
 
-        if project_entity:
-            if not self.server_caps.version or self.server_caps.version < (5, 4, 4):
-                raise ShotgunError("Per project schema operations require server "\
-                                   "version 5.4.4 or higher, server is %s" % (self.server_caps.version,))
-            else:
-                params["project"] = project_entity
+        params = self._add_project_param(params, project_entity)
```

Comment:
No worries at all - this was just a quick fix to get the tests working on older servers. Looks like you have done a much more holistic pass, trying to clean things up properly (which is totally awesome!). So please ignore/discard any of my changes in favour of your proper changes. This was just a quick fix to make sure we can release the API with all tests passing.